src/njoy21/input/BROADR.hpp

```diff
@@ -22,10 +22,20 @@ class BROADR {
       card4( is, this->card2.ntemp2, this->card2.istrap, this->card2.temp1 ){
   	Card5 card5( is );
   	if ( card5.mat1.value ){
+          if( card5.mat1.value < this->card2.mat1.value ){
+            Log::error("BROADR materials must be entered in ascending order." );
```

Comment:
Making this error statement the same as the next one makes it difficult to know which one threw the error. This statement could be a good `Log::info`, but we need more information so we know more specifically where the problem is.

Reply:
Any comments on this?